include/openssl/provider.h

```diff
@@ -22,8 +22,9 @@ int OSSL_PROVIDER_set_default_search_path(OSSL_LIB_CTX *, const char *path);
 
 /* Load and unload a provider */
 OSSL_PROVIDER *OSSL_PROVIDER_load(OSSL_LIB_CTX *, const char *name);
-OSSL_PROVIDER *OSSL_PROVIDER_try_load(OSSL_LIB_CTX *, const char *name,
-                                      int retain_fallbacks);
+OSSL_PROVIDER *OSSL_PROVIDER_multiple_load(OSSL_LIB_CTX *, const char *provid, const char *name);
+OSSL_PROVIDER *OSSL_PROVIDER_try_load(OSSL_LIB_CTX *, const char *provid,
```

Comment:
You cannot modify a signature of an existing API.
To add additional params you need a new _ex method.